src/tools.js

```diff
@@ -1,3 +1,69 @@
-exports.getGithubIssuesUrls = (repositories) => {
-    return repositories.map((repository) => `https://api.github.com/repos/${repository}/issues`);
+const Apify = require('apify');
+const { ISSUES_STATE, SUCCEEDED_STATUS, THIS_ACTOR_ID } = require('./constants');
+
+const { utils: { log } } = Apify;
+
+exports.getGithubIssuesRequests = (repositories, page = 1) => {
+    return repositories.map((repository) => ({
+        url: `https://api.github.com/repos/${repository}/issues?state=all&per_page=100&page=${page}`,
+        userData: { repository, page },
+    }));
 };
+
+exports.getModifiedIssues = async (currentIssues) => {
+    const modifiedIssues = {};
+
+    const lastRunIssues = await getLastRunIssues();
```

Comment:
It is better to store the past issues in named KV store
1. It is more flexible
2. Easily visible
3. Runs get deleted after retention